connectors/src/connectors/microsoft/temporal/spreadsheets.ts

```diff
@@ -165,11 +169,19 @@ async function processSheet(
   if (rawHeaders && rows.length > 1) {
     const headers = getSanitizedHeaders(rawHeaders);
 
+    const [, ...parents] = await getParents({
```

Comment:
Same why skipping the first element?

Reply:
`getParents` return itself as the first element (method name is confusing indeed, and we have the same for google and notion) - probably useful for `connector.retrieveContentNodeParents`, but here we only need the parents list

Reply:
Can we either create a function for it or add a parameter like `excludeSelf`?